### ISSUE TITLE
Error: listen EADDRINUSE: address already in use :::8000

### ISSUE THREAD

**Issue by lowrisk75** [oznu/docker-homebridge#162]
I keep having this error
Error: listen EADDRINUSE: address already in use :::8000

any idea how to change just the port 8000 and let docker with automatic port settings?

**Comment by Erik262** [comment]
Same problem here... first time it works, after some restarts no more.

**Comment by oznu** [comment]
This container requires you to run using host-networking, so automatic port settings are not applicable here.

Homebridge by default does not use port 8000, perhaps one of your plugins is though? I would need the full log to determine what's going on.

**Comment by synssins** [comment]
Update: Further troubleshooting reveals that failed installations of Homebridge or plugins within the root level OS result in a running process still listening on port 8000. I have killed the process, and Homebridge immediately came up with port 8000 working properly.

Found the process by typing: sudo netstat -pat | grep LISTEN
and looked for the port 8000.
Killed the process by typing: sudo kill (PID)
